modules/registrars/realtimeregister/src/Actions/Tlds/PricingSync.php

```diff
@@ -20,63 +20,62 @@ public function __invoke(Request $request)
     {
         $results = new ResultsList();
 
-        if (App::registrarConfig()->customerHandle() !== '') {
```

Comment:
This was added so we didn't get an error trying to sync while there was no customerdata set (yet). I don;t think it needs to be removed?